Return-Path: <q@uni.de>
Received: from localhost
	by localhost with POP3 (fetchmail-6.2.0)
	for netchild@localhost (single-drop); Sun, 16 Mar 2003 00:18:57 +0100 (CET)
Received: from mx2.freebsd.org (mx2.freebsd.org [216.136.204.119])
	by redbull.bpaserver.net (8.12.3/8.12.3) with ESMTP id h2FNAYEe046221
	for <Alex+FreeBSDcomitter@Leidinger.net>; Sun, 16 Mar 2003 00:10:34 +0100 (CET)
	(envelope-from q@uni.de)
Received: from hub.freebsd.org (hub.freebsd.org [216.136.204.18])
	by mx2.freebsd.org (Postfix) with ESMTP id 7269655A25
	for <Alex+FreeBSDcomitter@Leidinger.net>; Sat, 15 Mar 2003 15:10:33 -0800 (PST)
	(envelope-from q@uni.de)
Received: by hub.freebsd.org (Postfix)
	id 6DDF037B404; Sat, 15 Mar 2003 15:10:33 -0800 (PST)
Delivered-To: netchild@freebsd.org
Received: from mx1.FreeBSD.org (mx1.freebsd.org [216.136.204.125])
	by hub.freebsd.org (Postfix) with ESMTP
	id 6977337B401; Sat, 15 Mar 2003 15:10:33 -0800 (PST)
Received: from wrzx35.rz.uni-wuerzburg.de (wrzx35.rz.uni-wuerzburg.de [132.187.3.35])
	by mx1.FreeBSD.org (Postfix) with ESMTP
	id 22FA043F93; Sat, 15 Mar 2003 15:10:31 -0800 (PST)
	(envelope-from q@uni.de)
Received: from wrzx34.rz.uni-wuerzburg.de (wrzx34.rz.uni-wuerzburg.de [132.187.3.34])
	by wrzx35.rz.uni-wuerzburg.de (8.8.8/8.8.8/uniwue-MM-1.05) with ESMTP id AAA195143;
	Sun, 16 Mar 2003 00:10:29 +0100 (CET)
Received: from virusscan (localhost [127.0.0.1])
	by wrzx34.rz.uni-wuerzburg.de (Postfix) with ESMTP
	id 8EDD967CB1; Sun, 16 Mar 2003 00:10:29 +0100 (CET)
Received: from wrzx07.rz.uni-wuerzburg.de ([132.187.1.7])
        by virusscan (wrzx34 [132.187.3.34:25]) (amavisd-new) with ESMTP id 19257-01; Sun, 16 Mar 2003 00:10:29 +0100 (CET)
Received: from spamchecker (localhost [127.0.0.1])
	by wrzx07.rz.uni-wuerzburg.de (Postfix) with ESMTP
	id 810334672; Sun, 16 Mar 2003 00:10:28 +0100 (CET)
Received: from coyote.local (gb-007.galgenberg.net [132.187.222.7])
	by wrzx07.rz.uni-wuerzburg.de (Postfix) with ESMTP
	id 5DC5C466A; Sun, 16 Mar 2003 00:10:28 +0100 (CET)
Received: from roadrunner.local (roadrunner [192.168.0.147])
	by coyote.local (8.12.6/8.12.6) with ESMTP id h2FNATFd031287;
	Sun, 16 Mar 2003 00:10:29 +0100 (CET)
	(envelope-from q@roadrunner.local)
Received: from roadrunner.local (localhost [127.0.0.1])
	by roadrunner.local (8.12.8/8.12.8) with ESMTP id h2FNARt9059582;
	Sun, 16 Mar 2003 00:10:27 +0100 (CET)
	(envelope-from q@roadrunner.local)
Received: (from q@localhost)
	by roadrunner.local (8.12.8/8.12.8/Submit) id h2FNAQLg059581;
	Sun, 16 Mar 2003 00:10:26 +0100 (CET)
Date: Sun, 16 Mar 2003 00:10:26 +0100
From: Ulrich Spoerlein <q@uni.de>
To: Dan Naumov <dan.naumov@ofw.fi>
Cc: FreeBSD-gnats-submit@FreeBSD.org, netchild@FreeBSD.org
Subject: Re: ports/50024: games/quakeforge: zpak looks for /bin/bash and fails
Message-ID: <20030315231026.GC397@galgenberg.net>
References: <1047744651.51390.1.camel@localhost>
Mime-Version: 1.0
Content-Type: text/plain; charset=us-ascii
Content-Disposition: inline
In-Reply-To: <1047744651.51390.1.camel@localhost>
User-Agent: Mutt/1.4i
X-No-Archive: yes
X-Spam-Status: No, hits=-33.9 required=8.0
	tests=BAYES_30,IN_REP_TO,LOCAL_CLIENT,PATCH_UNIFIED_DIFF,
	      REFERENCES,USER_AGENT_MUTT
	version=2.50-string_20021002
X-Spam-Level: 
X-Spam-Checker-Version: SpamAssassin 2.50-string_20021002 (1.173-2003-02-20-exp)
X-Virus-Scanned: by amavisd-milter (http://amavis.org/)
X-Virus-Scanned: by amavisd-new (Rechenzentrum Universitaet Wuerzburg)
X-UIDL: ;oE"!`g$"!>S,!!&nV!!

It's actually a plain-sh script, so there is no need to add a bash
dependancy. The attached patch should work fine and fixes a minor quirk
with the script.


For the committer:

I said to maintain the port and Alexander put the following into
the commit log "Give maintainership to Ulrich Spoerlein <q@uni.de> from
ports." but it's still set to ports@freebsd.org

Please put the following as "patch-zpak" into games/quakeforge/files
Thanks.

--- tools/zpak.orig	Sat Mar 15 23:32:09 2003
+++ tools/zpak	Sat Mar 15 23:58:52 2003
@@ -1,4 +1,4 @@
-#! /bin/bash
+#!/bin/sh
 # create pak files with gzipped internals
 
 if [ $# -eq 0 ]; then
@@ -18,7 +18,7 @@
 
 	# cosmetic (gzip preserves timestamps.. could come in handy)
 	find * -type f \
-	-exec touch {} -r ${pakfile} \;
+	-exec touch -r ${pakfile} {} \;
 
 	find * -type f \
 	! -name '*.gz' ! -name '*.jpg' ! -name '*.ogg' ! -name '*.png' \

